src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -10806,8 +10806,14 @@ definitions:
           * mimetype (the mimetype of the content for a node)
           * encoding (the encoding of the content for a node)
           * category (a category for the node)
+          * tag (a tag that the node has)
           * aspect (an aspect that the node has)
         type: string
+      multi-value: |
+         Multiple value properties can exist for the following property types:
+          * text/mltext
+          * int/long/float/double (in Share only)
```

Comment:
I don't think we should mention Share. This is documenting the server - not the UI.